bases/rsptx/web2py_server/applications/runestone/controllers/admin.py

```diff
@@ -2448,7 +2448,7 @@ def add__or_update_assignment_question():
         try:
             activities_required = int(request.vars.get("activities_required"))
             if activities_required == -1:
-                activities_required = max(int(activity_count * 0.8), 1)
+                activities_required = max(round(activity_count * 0.8), 1)
```

Comment:
Oooh, this needs to go on my todo list, to port this to the assignment server.

Reply:
OK, I was not finding where this was used.  But I made a start anyway.  I'll keep it on a branch in case we need it.